pages/linux/schroot.md

```diff
@@ -0,0 +1,24 @@
+# schroot
+
+> Run command or start an interactive shell with a special root directory. More customizable than standard chroot.
```

Comment:
Perhaps this will read better?
```suggestion
> Run command or start an interactive shell with a different root directory. More customizable than `chroot`.
```